### DISCUSSION TITLE
Enforce UPPER_CASE for hardcoded `const`

### DISCUSSION THREAD

**Discussion by ydennisy** [eslint/eslint#14414] [Q&A]
There is a lint [rule](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md#enforce-that-all-const-variables-are-in-upper_case) for enforcing `UPPER_CASE` over all const values, however we want to enforce this rule only for hard coded values like in `const HARD_CODED = 1` is this possible?

**Comment by nzakas** [comment]
It is if you write a custom rule to do so. We don’t support that natively.

**Reply by ydennisy** [reply]
Ok cheers @nzakas

**Reply by Damian8610** [reply]
> Ok skål @nzakas

**Comment by Damian8610** [comment]
Tack
